crates/ibc/src/event.rs

```diff
@@ -60,6 +61,28 @@ pub mod types {
 #[repr(transparent)]
 pub struct IbcEventType(pub String);
 
+impl EventToEmit for IbcEvent {
+    const DOMAIN: &'static str = "ibc";
+}
+
+impl From<IbcEvent> for Event {
+    fn from(ibc_event: IbcEvent) -> Self {
+        let mut event = Self::new(
+            EventTypeBuilder::new_of::<IbcEvent>()
+                .with_segment(ibc_event.event_type.0)
+                .build(),
+            EventLevel::Tx,
```

Comment:
Do we emit the ibc events at the tx or block level?

Reply:
most are tx level events. pgf over ibc is at the block level, but idk a clean solution to change the event level there lol